javascript-carousel/main.js

```diff
@@ -0,0 +1,87 @@
+var count = 1;
```

Comment:
Change this to `var index = 0` instead of `var count = 1`, since you are only using `count` as it relates to an index. Then fix your other code to use your `index` variable instead.

Reply:
OK! Just did that.